dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml

```diff
@@ -0,0 +1,85 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
```

Comment:
Kik, what does it mean that I am the owner / who will receive notifications?

Reply:
Answered in the other PR.

Reply:
https://github.com/mozilla/dim/pull/42/files#r1028239357